bower.json

```diff
@@ -24,6 +24,7 @@
     "mithril": "~0.2.0",
     "mithril.postgrest": "https://github.com/catarse/mithril.postgrest.git#master",
     "moment": "~2.10.3",
-    "replace-diacritics": "https://github.com/yvg/js-replace-diacritics.git#master"
+    "replace-diacritics": "https://github.com/yvg/js-replace-diacritics.git#master",
+    "moment-timezone": "~0.4.0"
```

Comment:
I think we'll be able to solve the timezone problem without using the timezone plugin.